qtwebkit/Source/WebKit2/UIProcess/API/qt/qquickwebview.cpp

```diff
@@ -1350,6 +1351,37 @@ bool QQuickWebViewExperimental::firstFrameRendered() const
     return d->m_firstFrameRendered;
 }
 
+
+bool QQuickWebViewExperimental::repositionInputField() const
+{
+    Q_D(const QQuickWebView);
+    return d->m_repositionInputField;
+}
+
+void QQuickWebViewExperimental::setRepositionInputField(bool repositionInputField)
+{
+    Q_D(QQuickWebView);
+
+    if (d->m_repositionInputField == repositionInputField)
+        return;
+
+    d->m_repositionInputField = repositionInputField;
+    emit repositionInputFieldChanged();
+}
+
+void QQuickWebViewExperimental::focusEditableArea()
+{
+    Q_D(QQuickWebView);
+
```

Comment:
No, the purpose of the method is to trigger focus animation manually in case the automatic animation is turned off, so we don't want to turn the automatic animation on as a by product.